debian/jitsi-meet-tokens.postinst

```diff
@@ -39,7 +39,7 @@ case "$1" in
             echo "Application secret is mandatory"
         fi
         # Not allowed unix special characters in secret: /, \, ", ', `
-        if echo "$RET" | grep -q '[/\\\"\`]' ; then
+        if echo "$RET" | grep -q '[/\\\"\`]' || echo "$RET" | grep -q "'"; then
```

Comment:
```suggestion
        if echo "$RET" | grep -q '[/\\\"\`\']' ; then
```

Wouldn't this also do it?

Reply:
Yes, I've messed up, but it should be inside double quotes instead of single.